pkg/storage/loaders/model_plan_tracking_date_loader.go

```diff
@@ -0,0 +1,50 @@
+package loaders
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/graph-gophers/dataloader"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/mint-app/pkg/appcontext"
+)
+
+// GetModelPlanTrackingDateByModelPlanID uses a DataLoader to aggregate a SQL call and return the Model Plan Tracking Date in one query
+func (loaders *DataLoaders) GetModelPlanTrackingDateByModelPlanID(ctx context.Context, keys dataloader.Keys) []*dataloader.Result {
```

Comment:
I tried a different paradigm for the notification preferences loader. (`pkg/storage/loaders/user_notification_preferences_loader.go`). 

Effectively, I don't export the batch function, and I also create a single exported method which handles all the logic for loading the data loader (what you now have living in resolvers. I like that it keeps all responsibility for data loaders in a single place. What do you think? This comment is non-blocking, but if you agree with the paradigm, feel free to update as well!